Sources/VLCLibraryViewController.m

```diff
@@ -39,6 +40,20 @@
 
 @implementation EmptyLibraryView
 
+- (id)initWithFrame:(CGRect)frame isInFolder:(bool)inFolder
```

Comment:
The inFolder bool can go, we won't have folders anymore

Reply:
Can I go ahead and delete all localized strings for `FOLDER_EMPTY` and `FOLDER_EMPTY_LONG`?

Reply:
absolutely